tool/flutter_ci_script_stable.sh

```diff
@@ -26,8 +26,8 @@ declare -ar PROJECT_NAMES=(
     "ios_app_clip"
     "isolate_example"
     "jsonexample"
-    # TODO: Re-enable once Dart 2.17 is stable
-    # "material_3_demo"
+    # TODO(domesticmouse): Re-enable once Dart 2.17 is stable
+    # "material_you_demo"
```

Comment:
This should be the directory that contains the sample. So, please revert to `material_3_demo`.

Reply:
I will change the directory name to material_you_demo and make sure the name matches. Sorry for the confusion.

Reply:
My understanding is that they are using Material 3 for the design changes, while Material You is more about the customization features. Perhaps we should just lave this as material_3_demo? @guidezpl do you have any opinion on this?

Reply:
That's right, prefer to use Material 3 for the name and directory. Material You is a term that kind of stuck because it was announced first on Android.